### PR TITLE
Correct when will-change forms a containing block, and simplify the l…

### PR DESCRIPTION
**Pull request by aniederer-chatham**
…ist for absolute elements

### Description

From CSS Will Change Module Level 1:

> If any non-initial value of a property would cause the element to generate a containing block for absolutely positioned elements, specifying that property in will-change must cause the element to generate a containing block for absolutely positioned elements.
> 
> If any non-initial value of a property would cause the element to generate a containing block for fixed positioned elements, specifying that property in will-change must cause the element to generate a containing block for fixed positioned elements.
> 

I have personally tested this behavior in Chrome, Firefox, and Safari, and they all comply (except for container-type for some reason, maybe that's a bug?)

### Motivation

Current docs appear to be incorrect about cross-browser behavior, and will-change values other than transform/perspective/filter. 

### Additional details

Just below this:

https://drafts.csswg.org/css-will-change/#example-7225dcbc

### CODE REVIEW

File: files/en-us/web/css/containing_block/index.md [hunk 1]
```diff
@@ -34,12 +34,10 @@ The process for identifying the containing block depends entirely on the value o
 3. If the `position` property is **`fixed`**, the containing block is established by the {{glossary("viewport")}} (in the case of continuous media) or the page area (in the case of paged media).
 4. If the `position` property is **`absolute`** or **`fixed`**, the containing block may also be formed by the edge of the _padding box_ of the nearest ancestor element that has any of the following:
 
-   1. A {{cssxref("transform")}} or {{cssxref("perspective")}} value other than `none`
-   2. A {{cssxref("will-change")}} value of `transform` or `perspective`
-   3. A {{cssxref("filter")}} value other than `none` or a `will-change` value of `filter` (only works on Firefox)
-   4. A {{cssxref("contain")}} value of `layout`, `paint`, `strict` or `content` (e.g. `contain: paint;`)
-   5. A {{cssxref("container-type")}} value other than `normal`
-   6. A {{cssxref("backdrop-filter")}} other than `none` (e.g. `backdrop-filter: blur(10px);`)
+   1. A {{cssxref("filter")}}, {{cssxref("backdrop-filter")}}, {{cssxref("transform")}}, or {{cssxref("perspective")}}, value other than `none`
```
Comment (chrisdavidmills) [inline files/en-us/web/css/containing_block/index.md:37]:
```suggestion
   1. A {{cssxref("filter")}}, {{cssxref("backdrop-filter")}}, {{cssxref("transform")}}, or {{cssxref("perspective")}} value other than `none`.
```

File: files/en-us/web/css/containing_block/index.md [hunk 2]
```diff
@@ -34,12 +34,10 @@ The process for identifying the containing block depends entirely on the value o
...
+   2. A {{cssxref("contain")}} value of `layout`, `paint`, `strict` or `content` (e.g. `contain: paint;`)
```
Comment (chrisdavidmills) [inline files/en-us/web/css/containing_block/index.md:38]:
```suggestion
   2. A {{cssxref("contain")}} value of `layout`, `paint`, `strict` or `content` (e.g. `contain: paint;`).
```

File: files/en-us/web/css/containing_block/index.md [hunk 3]
```diff
@@ -34,12 +34,10 @@ The process for identifying the containing block depends entirely on the value o
...
+   3. A {{cssxref("container-type")}} value other than `normal`
```
Comment (chrisdavidmills) [inline files/en-us/web/css/containing_block/index.md:39]:
```suggestion
   3. A {{cssxref("container-type")}} value other than `normal`.
```

File: files/en-us/web/css/containing_block/index.md [hunk 4]
```diff
@@ -34,12 +34,10 @@ The process for identifying the containing block depends entirely on the value o
...
+   4. A {{cssxref("will-change")}} value containing a property for which a non-initial value would form a containing block (e.g. `filter`, `transform`)
```
Comment (chrisdavidmills) [inline files/en-us/web/css/containing_block/index.md:40]:
```suggestion
   4. A {{cssxref("will-change")}} value containing a property for which a non-initial value would form a containing block (e.g. `filter` or `transform`).
```

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Hello @aniederer-chatham, and thank you for taking the time to make this contribution to MDN. This looks good to me; I just had a few minor formatting issues for you to clear up.

**Comment by aniederer-chatham** [comment]
Thanks, added those changes!

**Review by chrisdavidmills — APPROVED** [review]
@aniederer-chatham perfect, thanks again for the contribution. Let's get this merged!

**Pull request merged**